### ISSUE TITLE
[Safe Mode] Unclean shutdown detected! Windows 11

### ISSUE THREAD

**Issue by kitenski** [obsproject/obs-studio#10833]
### Operating System Info

Windows 11

### OBS Studio Version

30.1.2

### OBS Studio Log URL

https://obsproject.com/logs/ddtpl0TX1ucKOGm9

### Expected Behavior

OBS to start normally after a clean shutdown

### Current Behavior

Every time I start my desktop I get a Unclean shutdown message and have to elect to start normally. When I finish using my desktop I always chose Shut Down from the start menu

### Steps to Reproduce

1. Start Windows 11
2. Use OBS
3. Cleanly shutdown Windows 11
...

**Comment by RytoEX** [comment]
Did you restart your PC while OBS was running?

**Comment by kitenski** [comment]
> Did you restart your PC while OBS was running?

No, just do a shutdown from the start menu, after I've stopped recording.

**Comment by RytoEX** [comment]
Did you Exit or Close OBS itself before shutting down Windows?

**Comment by Fenrirthviti** [comment]
Please check for OBS crash logs on shutdown, or other Windows Events that might indicate something not exiting cleanly. Without more information on what is causing this, this is not actionable.

For posterity, the log file currently provided shows a clean shutdown.

**Comment by Senliast** [comment]
I have the same problem. I attach the files from unclean shutdown (2024-08-07) and clean shutdown (2024-08-15). The difference i notice between these two is, that unclean shutdown ends with "All scene data cleared", and the clean one has other actions after it, the "web-server" is shutting down and so on. Another one (2024-08-16) didnt shutdown yet, i just attached it as proof, cuz there is not "unclean shutdown" message at start. So actually, sometimes it shuts down correct, but most times - not. I cannot tell any regularity.

I had old version of OBS as these log files were collected, but now i updated to the latest one (30.2.3), tested - and its the same, the issue still persists. There are no crashlogs from OBS in Windows Event Viewer.

[2024-08-07 21-22-37.txt](https://github.com/user-attachments/files/16639740/2024-08-07.21-22-37.txt)
[2024-08-15 23-59-34.txt](https://github.com/user-attachments/files/16639741/2024-08-15.23-59-34.txt)
[2024-08-16 18-43-50.txt](https://github.com/user-attachments/files/16639742/2024-08-16.18-43-50.txt)

**Comment by RytoEX** [comment]
> I have the same problem. I attach the files from unclean shutdown (2024-08-07) and clean shutdown (2024-08-15). The difference i notice between these two is, that unclean shutdown ends with "All scene data cleared", and the clean one has other actions after it, the "web-server" is shutting down and so on. Another one (2024-08-16) didnt shutdown yet, i just attached it as proof, cuz there is not "unclean shutdown" message at start. So actually, sometimes it shuts down correct, but most times - not. I cannot tell any regularity.
> 
> I had old version of OBS as these log files were collected, but now i updated to the latest one (30.2.3), tested - and its the same, the issue still persists. There are no crashlogs from OBS in Windows Event Viewer.
> 
> [2024-08-07 21-22-37.txt](https://github.com/user-attachments/files/16639740/2024-08-07.21-22-37.txt) [2024-08-15 23-59-34.txt](https://github.com/user-attachments/files/16639741/2024-08-15.23-59-34.txt) [2024-08-16 18-43-50.txt](https://github.com/user-attachments/files/16639742/2024-08-16.18-43-50.txt)

Any unclean shutdown can cause this alert. It may not be the same as OP.

The first log shows that you chose to launch OBS normally instead of in Safe Mode. Then OBS crashed or was force quit during application shutdown.

The second log shows a completely normal OBS shutdown.

The third log shows either a partial log (the OBS session is still active) or an abnormal shutdown. It's impossible to tell without more context.

If OBS shuts down normally, then the log will end with `Number of memory leaks: 0`. If the OBS log does not end with that, then the shutdown was almost certainly abnormal and interrupted by something (Windows shutdown, task/process ended/killed, application crash). If OBS crashes, there _should_ be a crash log produced in `%appdata%\obs-studio\crashes`.

My questions would be:
1. Are you exiting OBS normally (File -> Exit, Exit button, X button)?
2. Have you tried removing or updating `logi_obs_plugin_x64.dll`?
3. Have you tried removing or disabling any active/loaded scripts?
4. Have you experienced an abnormal shutdown while OBS was running in Safe Mode?

**Comment by Senliast** [comment]
@RytoEX i have the same problem as OP, because this happens only after i shutdown or reboot PC while OBS is running, on next start of PC. And that's what OP is saying. It does not happen, if i close OBS and start again. But this is exactly how i use OBS, because i am using it as ShadowPlay replacement - it auto-starts and runs in background, then a script that i wrote automatically launches or stops the replay buffer, if certain games are launched. And this problem was not present on Win 10, only after i installed Win 11. Therefore i am sure, it has nothing to do with scripts, OBS just has a problem with Win 11 shutdown. In the '%appdata%\obs-studio\crashes' folder, there are only old crash logs, no recent ones. I have no idea how to update '%appdata%\obs-studio\crashes', i will attach properties of the file as screenshot.

![PXL_20240816_214007694](https://github.com/user-attachments/assets/ee2269c5-6bf8-4064-ad2a-eab64ed67894)

**Comment by kitenski** [comment]
> Please check for OBS crash logs on shutdown, or other Windows Events that might indicate something not exiting cleanly. Without more information on what is causing this, this is not actionable.
> 
> For posterity, the log file currently provided shows a clean shutdown.

Apologies I missed this reply, I still get this issue, logs still show nothing out of the ordinary! PC runs fine, no other error messages or Windows events that I can see!
The only event that shows as an Error in Event Viewer is:

The Secure Boot update failed to update a Secure Boot variable with error Secure Boot is not enabled on this machine.. For more information, please see https://go.microsoft.com/fwlink/?linkid=2169931

However I *DO* see this in the application logs

Event 10001, Winsrv
The following application attempted to veto the shutdown: obs64.exe.

```
- System 

  - Provider 

   [ Name]  Microsoft-Windows-Winsrv 
   [ Guid]  {9d55b53d-449b-4824-a637-24f9d69aa02f} 
 
   EventID 10001 
 
   Version 0 
 
   Level 4 
 
   Task 0 
 
   Opcode 0 
 
   Keywords 0x8000000000000000 
 
  - TimeCreated 

   [ SystemTime]  2024-08-19T17:13:23.4777216Z 
 
   EventRecordID 37834 
 
   Correlation 
 
  - Execution 

   [ ProcessID]  940 
   [ ThreadID]  2624 
 
   Channel Application 
 
   Computer garagePC 
 
  - Security 

   [ UserID]  S-1-5-18 
 

- UserData 

  - VetoAppEvent 

   AppName obs64.exe 
 
   ResponseTime 0 
```
 
 


Logs from last night and this morning where I got the same issue attached.

[2024-08-20 13-29-07.txt](https://github.com/user-attachments/files/16675788/2024-08-20.13-29-07.txt)
[2024-08-19 17-28-24.txt](https://github.com/user-attachments/files/16675789/2024-08-19.17-28-24.txt)
